operator/internal/manifests/internal/config/loki-config.yaml

```
@@ -222,6 +232,46 @@ limits_config:
   shard_streams:
     enabled: true
     desired_rate: {{ . }}MB
+{{- end }}
+{{- with .Stack.Limits.Global.OTLP }}
+  otlp_config:
+    {{- with .ResourceAttributes }}
+    resource_attributes:
+      ignore_defaults: {{ .IgnoreDefaults }}
+      {{- with .Attributes}}
+      attributes_config:
+      {{- range . }}
+      - action: {{ .Action }}
+        attributes:
+        {{- range .Attributes }}
+        - {{ . }}
+        {{- end}}
+        regex: {{ .Regex }}
```

Reorder to have the variable-length part last and also make `regex` optional (avoid `regex: ` for an empty value of `.Regex`):

```suggestion
      - action: {{ .Action }}
        {{- with .Regex }}
        regex: {{ . }}
        {{- end }}
        attributes:
        {{- range .Attributes }}
        - {{ . }}
        {{- end}}
```